src/plugins/data/common/index_patterns/index_patterns/index_patterns.ts

```diff
@@ -454,16 +454,18 @@ export class IndexPatternsService {
   }
 
   /**
-   * Create a new index pattern and save it right away
+   * Create a new index pattern and save it right away and make the default
```

Comment:
IMO, this doesn't need to be changed. Its only made default if its the first index pattern being created.

Reply:
Reverted.